src/commands/game_commands/hint.ts

```diff
@@ -20,6 +20,10 @@ function isHintMajority(message: GuildTextableMessage, gameSession: GameSession)
         return gameSession.gameRound.getHintRequests() >= Math.floor(eliminationScoreboard.getAlivePlayersCount() * 0.5) + 1;
     }
 
+    if (gameSession.isMultipleChoiceMode()) {
```

Comment:
Remove this for consistency?

Reply:
It is consistent with elimination, since only alive players can request hints